app/Commands/Sites/CreateCommand.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace App\Commands\Sites;
+
+use App\Commands\BaseCommand;
+use App\Commands\Concerns\HasServerIdParameter;
+use App\Helpers\OptionsHelper;
+use App\Questions\Ask;
+use App\Questions\Choice;
+use App\Questions\Confirm;
+use App\Questions\HasQuestions;
+use Illuminate\Support\Str;
+
+class CreateCommand extends BaseCommand
+{
+    use HasQuestions;
+    use HasServerIdParameter;
+
+    protected $signature = 'sites:create
+                            {installation_method : Type of installation (wp or blank)}
+                            {server_id? : Server ID}
+                            {--domain= : Domain name}
+                            {--site_user=}
+                            {--db_name=}
+                            {--db_user=}
+                            {--db_pass=}
+                            {--wp_title=}
+                            {--wp_admin_user=}
+                            {--wp_admin_email=}
+                            {--wp_admin_pass=}
+                            {--php_version=}
+                            {--page_cache_enabled}
+                            {--https_enabled}
+                            {--profile=}
+                            {--f|force}';
+
+    protected $description = 'Create a site';
+
+    protected array $userInput;
+
+    protected function action(): int
+    {
+        if (!in_array($this->argument('installation_method'), OptionsHelper::INSTALLATION_METHODS, true)) {
+            $this->error('Invalid site type.');
+            $this->newLine(1);
+            return self::INVALID;
+        }
+
+        $server = $this->selectServer('deploy to')->first();
+
+        if (is_null($server)) {
+            return self::INVALID;
+        }
+
+        $this->userInput['domain'] = Ask::make('Domain')
+            ->nonInteractive($this->nonInteractive())
+            ->resolveAnswer($this);
+
+        $this->userInput += $this->askQuestions($this->nonInteractive());
+
+        $site = $this->spinupwp->createSite($server->id, array_merge($this->arguments(), $this->options(), $this->userInput));
+
+        $this->successfulStep("{$site->domain} is {$site->status} (event_id = {$site->eventId()})");
```

Comment:
This should be consistent will other command outputs and let's show the passwords here:

```
==> Site queued for creation.
    Event ID    Site          Database Password    WordPress Admin Password
    1180        domain.com    dk3kskfks            dsfdsf3sd
```

Reply:
✅